### PR TITLE
Create question plan

### LINKED ISSUE #20: Create Question Plan
Need a document for the questions being asked. 
- Split by case
- List of associated answers, and the correct one
- Some details on the topic being covered

### PR DESCRIPTION
**Pull request by Alice-Jaffray**
**Description of work.**
Created questions for all the levels

**To test:**
Check for spelling and grammar

Fixes #20

### CODE REVIEW

File: docs/planning/QuestionPlan.md [hunk 1]
```diff
@@ -52,4 +52,167 @@
...
+    - *= Attack*
+    - *= Vulnerability*
+    - *= Threat*
+
+- "We need to get control back over this house for Ginny, what's the definition of control?"
+    - *= Used to remove or limit the vulnerability*
+    - *= Attempt to exploit a vulnerability*
+    - *= Circumstances or events that could potentially lead to harm or loss*
+
+- "I know there are 4 types of attackers, but what are they again?"
```
Comment (cailafinn) [inline docs/planning/QuestionPlan.md:78]: There are actually more than this, these are just some examples. This is probably fine, but Jason might mention it

File: docs/planning/QuestionPlan.md [hunk 2]
```diff
@@ -52,4 +52,167 @@
...
+    - *= Amateurs, Crackers, Hacktivists, Terrorists*
+    - *= Amateurs, Biscuits, Hacktivists, Terrorists*
+    - *= Amateurs, Cracker, Lacktivists, Terrorists*
+
+- "Security is so hard to get right, which of the following is a reason why?"
+    - *= Arms race, Weakest links, resistance against overheads*
+    - *= Arms wrestle, Weakest links, attacks "outside the box"*
+    - *= You are the weakest link, goodbye*
+
+- "I need to start thinking about which type of security attack occurred in this house, which of the following types of security attacks?"
```
Comment (cailafinn) [inline docs/planning/QuestionPlan.md]:
```suggestion
- "I need to start thinking about which type of security attack occurred in this house, which of the following are types of security attacks?"
```

File: docs/planning/QuestionPlan.md [hunk 3]
```diff
@@ -52,4 +52,167 @@
...
+
+- "This clue shows I need to think about the security risk management lifecycle - select the correct lifecycle"
+    - *= Identify -> Analyse -> Threat -> Monitor*
+    - *= Threat -> Analyse -> Identify -> Monitor*
+    - *= Identify -> Monitor -> Threat -> Analyse*
+
+- "I'm going to need to make security risk statements - but what are they?"
+    - *= a method of presenting related information in the expression of a security risk*
+    - *= a comprehensive process that requires organizations to frame risk assess risk; respond to risk once determined; monitor risk on an ongoing basis*
+    - *= Have I mentioned apple bobbing before?*
```
Comment (cailafinn) [inline docs/planning/QuestionPlan.md:147]: nice

File: docs/planning/QuestionPlan.md [hunk 4]
```diff
@@ -52,4 +52,167 @@
...
+    - *= Use of experts, allows focus on high-priority usability principles*
+
+- "What are some examples of expert evaluation I can give to Ginny?"
+    - *= cognitive walkthrough & heuristic evaluations*
+    - *= laboratory-based user testing & questionnaires*
+    - *= interviews & observing users (one-way mirrors)*
+
+- "This clue is to do with usability features. Which option describes usability features"
+    - *= Usability is a central concept to HCI, which considers how easy a system is to use.*
+    - *= Usability is a central concept to HCI, which considers how difficult a system is to use.*
```
Comment (cailafinn) [inline docs/planning/QuestionPlan.md]: These are two sides of the same coin. "Not difficult" as an answer to the second question is too similar to "easy" for the first one

**Review by cailafinn — CHANGES_REQUESTED** [review]
Just a few notes, but otherwise looks really good!

File: docs/planning/QuestionPlan.md (hunk 4 above)
Comment (Alice-Jaffray) [reply docs/planning/QuestionPlan.md]: That was the idea to make them look for the specifics but I've changed it 👍

**Review by cailafinn — APPROVED** [review]
Looks good to me. Thanks for doing this :)

**Pull request merged**